core/deps/CMakeLists.txt

```diff
@@ -84,5 +84,9 @@ add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/alfons)
 ###############
 set(SQLITECPP_RUN_CPPLINT OFF CACHE BOOL "")
 set(SQLITECPP_RUN_CPPCHECK OFF CACHE BOOL "")
+set(SQLITE_ENABLE_COLUMN_METADATA OFF CACHE BOOL "")
 
 add_subdirectory(SQLiteCpp)
+
+# Extensions aren't needed for MBTiles and aren't available in older versions of sqlite3.
+target_compile_definitions(SQLiteCpp PRIVATE SQLITE_OMIT_LOAD_EXTENSION)
```

Comment:
We can probably omit more here, but I could not figure out what might be best set here from giving a quick look at https://www.sqlite.org/compile.html.

Probably can also use `SQLITE_OMIT_DEPRECATED`